inlang/source-code/sdk2/src/project/util/toMessageV1.test.ts

```diff
@@ -32,21 +32,25 @@ const messageV1: MessageV1 = {
 	selectors: [],
 };
 
-const bundle: BundleNested = {
-	alias: {},
-	id: "hello_world",
+const humanReadableId = "awful_lamb_mend_smooth";
+
+const bundle = {
+	alias: {
+		default: "hello_world",
```

Comment:
Shall we remove the concept of `default` alias? 

The pattern of `default` will break latest with multiple importers. I think it's best to use the plugin key as alias key `{ 'i18next': 'hello_world' }`. There are really only two scenarios: 

1. Multiple aliases exist (default doesn't make sense. Worst of all, default might be overwritten or change conflicts are generated)
2. One alias exists (no default needed. app can derive the alias via `alias[Object.keys(alias)[0]]`)

--- 

For context, we introduced `alias.default` to avoid a breaking change in Paraglide JS (cc @LorisSigrist). Paraglide JS would be able to fallback to the default alias https://github.com/opral/monorepo/commit/68edada6fc45dbaaaf4d2ad13505396c73abe790. As shown though, the concept of a default alias doesn't need to exist for Paraglide to fallback to an alias. 

I even think Paraglide doesn't need a concept of default alias at all to avoid breaking changes. Any alias should be compiled as long as it is a valid JS identifier. Why only compile the `default` alias? Imagine that someone wants to migrate from i18next to Paraglide JS. If Paraglide JS compiles the `i18next` aliases (that are a valid js identifier), the migration is as simple as 

```diff
- t("my_header_button")
+ m.my_header_button()
```

In a second step, Paraglide JS would signal that the bundle id should be used directly. But, a dev can in the first step just use the i18next (or any other JS valid) alias. 

```diff
-m.my_header_button()
+m.blue_moon_shine_world()
```

Reply:
Open a dedicated issue for that to not block this PR? 

![CleanShot 2024-08-07 at 12 34 29@2x](https://github.com/user-attachments/assets/01a9a3d3-e7da-4fe0-b6f3-e1ec0d67ecb3)

Also, removing `alias.default` is yet another thing we will not have to explain then. The question came up from someone in an issue already (can't find the link)

![CleanShot 2024-08-07 at 12 24 30@2x](https://github.com/user-attachments/assets/2e38ac04-64ce-46ec-af34-841e131e6875)

Reply:
100% agree 
- ~~lint~~ **validation**rules for the rescue - they will ensure duplicates independent of the alias. 
- if aliases of different exporters/importers match paraglide dedubs them if they differe it makes sense to have for each entry a message function.